examples/model_compress/pruning/taylorfo_transformers_evaluator.py

```diff
@@ -0,0 +1,62 @@
+from datasets import load_dataset
+
+dataset = load_dataset('yelp_review_full')
+
+from transformers import AutoTokenizer
+
+tokenizer = AutoTokenizer.from_pretrained('bert-base-cased')
+
+def tokenize_function(examples):
+    return tokenizer(examples['text'], padding='max_length', truncation=True)
+
+tokenized_datasets = dataset.map(tokenize_function, batched=True)
+
+small_train_dataset = tokenized_datasets['train'].shuffle(seed=42).select(range(1000))
+small_eval_dataset = tokenized_datasets['test'].shuffle(seed=42).select(range(1000))
+
+from transformers import AutoModelForSequenceClassification
+
+model = AutoModelForSequenceClassification.from_pretrained('bert-base-cased', num_labels=5)
+
+from transformers import TrainingArguments
+
+training_args = TrainingArguments(output_dir='test_trainer')
+
+import numpy as np
+from datasets import load_metric
+
+metric = load_metric('accuracy')
+
+def compute_metrics(eval_pred):
+    logits, labels = eval_pred
+    predictions = np.argmax(logits, axis=-1)
+    return metric.compute(predictions=predictions, references=labels)
+
+from transformers import TrainingArguments, Trainer
```

Comment:
suggest to put these `import`s at the top, which would be more formal

Reply:
moved